4-Deployment/1-deploy-storage/SPA/src/App.jsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React, { useState } from "react";
+
+import { MsalProvider, AuthenticatedTemplate, UnauthenticatedTemplate, useMsal } from "@azure/msal-react";
+
+import { PageLayout, ProfileData } from "./ui.jsx";
+
+import Button from "react-bootstrap/Button";
+import "./styles/App.css";
+
+const ProfileContent = () => {
+    /**
+     * useMsal is hook that returns the PublicClientApplication instance, 
+     * an array of all accounts currently signed in and an inProgress value 
+     * that tells you what msal is currently doing. For more, visit: 
+     * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/hooks.md
+     */
+    const { instance, accounts, inProgress } = useMsal();
+    const account = useAccount(accounts[0] || {});
+    const [graphData, setGraphData] = useState(null);
+
+    /**	
+     * We recommend that your app calls the acquireTokenSilent API on your PublicClientApplication 	
+     * object each time you need an access token to access an API.	
+     */
+    RequestProfileData = () => {
+        instance.acquireTokenSilent({
+            scopes: protectedResources.graphMe.scopes,
```

Comment:
Uncaught ReferenceError: protectedResources is not defined. Missing import:

import { protectedResources } from "./authConfig";